docs/example/vllm/build/ray_init.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -e
+subcommand=$1
+shift
+
+ray_port=6379
+ray_init_timeout=300
+
+case "$subcommand" in
+  worker)
+    ray_address=""
+    while [ $# -gt 0 ]; do
+      case "$1" in
+        --ray_address=*)
+          ray_address="${1#*=}"
+          ;;
+        --ray_port=*)
+          ray_port="${1#*=}"
+          ;;
+        *)
+          echo "unknown argument: $1"
+          exit 1
+      esac
+      shift
+    done
+
+    if [ -z "$ray_address" ]; then
+      echo "Error: Missing argument --ray_address"
+      exit 1
+    fi
+
+    for (( i=0; i < $ray_init_timeout; i+=5 )); do
+      ray start --address=$ray_address:$ray_port --block
+      if [ $? -eq 0 ]; then
+        echo "Worker: Ray runtime started with head address $ray_address:$ray_port"
+        exit 0
+      fi
+      echo "Wait the ray worker to be active..."
+      sleep 5s;
+    done
+    echo "Ray worker starts timeout, head address: $ray_address:$ray_port"
+    exit 1
+    ;;
+
+  head)
```

Comment:
This is a great example, thanks :pray:  
One minor suggestion: Although an example, it is good to be in line with the terminology of `leader` and `worker`, as follows, 
```suggestion
  leader)
```